frontend/public/components/masthead-toolbar.jsx

```diff
@@ -408,6 +415,11 @@ class MastheadToolbarContents_ extends React.Component {
         {
           component: <GuidedTourMastheadTrigger ref={tourRef} />,
         },
+        {
+          label: t('public~ShareFeedback'),
```

Comment:
When you add the string to the t function you should add it as it appears in english ie: 'Share Feedback'. Then you run yarn run i18n in front end to generate the json files needed for translation.

Reply:
ah okay thank you joe!